thorlcr/master/thmastermain.cpp

```diff
@@ -461,7 +460,7 @@ class CThorEndHandler : public CSimpleInterface, implements IThreaded
     void start(unsigned timeoutSecs)
     {
         bool expected = false;
-        if (started.compare_exchange_strong(expected, true));
+        if (started.compare_exchange_strong(expected, true))
```

Comment:
this looks like a bug fix.  Should it also be in a separate jira/PR? same for the line below.

Reply:
it's in another PR already, but wasn't upmerged (but is now), will disappear when I rebase.

Reply:
@ghalliday now rebased